### DISCUSSION TITLE
OptionsPicker should open with the last selected entry

### DISCUSSION THREAD

**Discussion by michaelkebe** [grafana/grafana#70885] [Dashboards, Panels & Visualizations]
Often I want to check a bunch of hosts with a grafana dashboard (e.g. node exporter). The dashboard has a host variable filled via the query type label values.

When I select a host in the dashboard and then want to select the next host I am expecting that the dropdown menu of the host variable remembers my last selection, so I can easily use the keyboard down arrow to select the next one.

But the dropdown opens always with nothing selected. Pressing the down arrow selects the first one of the listed options.

### Current behaviour
* Selecting host 7
![image](https://github.com/grafana/grafana/assets/74952/c71425b3-130a-4402-9cc8-6d7b488c1aa0)

* Reopening the drop down
![image](https://github.com/grafana/grafana/assets/74952/256900a4-ecaa-4752-9c37-3ea39726d61d)
![image](https://github.com/grafana/grafana/assets/74952/011d1e44-c4b6-47ed-9305-067e3ca7f081)

### Wanted behaviour
* Selecting host 7
![image](https://github.com/grafana/grafana/assets/74952/c71425b3-130a-4402-9cc8-6d7b488c1aa0)

* Reopening the drop down
![image](https://github.com/grafana/grafana/assets/74952/256900a4-ecaa-4752-9c37-3ea39726d61d)
* The last selected host 7 is already selected so I can move the selection with the arrow keys
![image](https://github.com/grafana/grafana/assets/74952/4a01d6a9-d3e4-43fa-b491-6900127c8ed2)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82592. Feel free to continue the discussion around this there. Thank you!
